lab_1/main.py

```diff
@@ -12,9 +24,23 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+    tokens = []
+    if type(text) == str:
```

Comment:
`type` is considered a bad style, look for smth else.
e.g. `isinstance`

Reply:
ok, thanks